Gemfile.lock

```diff
@@ -126,6 +126,3 @@ DEPENDENCIES
   rspec-puppet
   sshkey (= 1.7.0)
   webmock (~> 1.20.0)
-
-BUNDLED WITH
-   1.10.5
```

Comment:
The removal shouldn't or the BUNDLED WITH shouldn't?  (it's largely been backed out from bundler)